`ERR_PNPM_NO_SCRIPT  Missing script: cloudflare`

Followed installation instructions for Cloudflare per the readme:

`pnpm add -D wrangler@latest @opennextjs/cloudflare`

`pnpx cloudflare` results in `ERR_PNPM_NO_SCRIPT  Missing script: cloudflare`

I can verify that both the `@opennextjs/cloudflare` folder is present in `node_modules` and cloudflare is not in `node_modules/.bin`

Node v20
pnpm v9.1.0

Thank you all for your hard work on this, can't wait to run more sites on Cloudflare :)

--- a/packages/cloudflare/README.md
+++ b/packages/cloudflare/README.md
@@ -16,6 +16,8 @@ You can use [`create-next-app`](https://nextjs.org/docs/pages/api-reference/cli/
   npm add -D wrangler@latest @opennextjs/cloudflare
   # or
   pnpm add -D wrangler@latest @opennextjs/cloudflare
+  # or
+  yarn add -D wrangler@latest @opennextjs/cloudflare
   ```
 
 - add a `wrangler.toml` at the root of your project
@@ -40,6 +42,8 @@ You can enable Incremental Static Regeneration ([ISR](https://nextjs.org/docs/ap
   npx wrangler kv namespace create NEXT_CACHE_WORKERS_KV
   # or
   pnpm wrangler kv namespace create NEXT_CACHE_WORKERS_KV
+  # or
+  yarn wrangler kv namespace create NEXT_CACHE_WORKERS_KV
   ```
 
 - Paste the snippet to your `wrangler.toml`:
@@ -67,6 +71,8 @@ Run the following commands to preview the production build of your application l
   npx cloudflare
   # or
   pnpm cloudflare
+  # or
+  yarn cloudflare
   ```
 
 - Preview the app in Wrangler
@@ -75,6 +81,8 @@ Run the following commands to preview the production build of your application l
   npx wrangler dev
   # or
   pnpm wrangler dev
+  # or
+  yarn wrangler dev
   ```
 
 ## Deploy your app
@@ -87,4 +95,6 @@ Deploy your application to production with the following:
   npx cloudflare && npx wrangler deploy
   # or
   pnpm cloudflare && pnpm wrangler deploy
+  # or
+  yarn cloudflare && yarn wrangler deploy
   ```

--- a/packages/cloudflare/README.md
+++ b/packages/cloudflare/README.md
@@ -13,6 +13,8 @@ You can use [`create-next-app`](https://nextjs.org/docs/pages/api-reference/cli/
 - add the following `devDependencies` to the `package.json`:
 
   ```bash
+  npm add -D wrangler@latest @opennextjs/cloudflare
+  # or
   pnpm add -D wrangler@latest @opennextjs/cloudflare
   ```
 
@@ -36,6 +38,8 @@ You can enable Incremental Static Regeneration ([ISR](https://nextjs.org/docs/ap
 
   ```bash
   npx wrangler kv namespace create NEXT_CACHE_WORKERS_KV
+  # or
+  pnpm wrangler kv namespace create NEXT_CACHE_WORKERS_KV
   ```
 
 - Paste the snippet to your `wrangler.toml`:
@@ -60,12 +64,16 @@ Run the following commands to preview the production build of your application l
 - build the app and adapt it for Cloudflare
 
   ```bash
-  pnpx cloudflare
+  npx cloudflare
+  # or
+  pnpm cloudflare
   ```
 
 - Preview the app in Wrangler
 
   ```bash
+  npx wrangler dev
+  # or
   pnpm wrangler dev
   ```
 
@@ -76,9 +84,7 @@ Deploy your application to production with the following:
 - build the app and adapt it for Cloudflare
 
   ```bash
-  pnpx cloudflare
-  ```
-
-  ```bash
-  pnpm wrangler deploy
+  npx cloudflare && npx wrangler deploy
+  # or
+  pnpm cloudflare && pnpm wrangler deploy
   ```

--- a/packages/cloudflare/README.md
+++ b/packages/cloudflare/README.md
@@ -18,6 +18,8 @@ You can use [`create-next-app`](https://nextjs.org/docs/pages/api-reference/cli/
   pnpm add -D wrangler@latest @opennextjs/cloudflare
   # or
   yarn add -D wrangler@latest @opennextjs/cloudflare
+  # or
+  bun add -D wrangler@latest @opennextjs/cloudflare
   ```
 
 - add a `wrangler.toml` at the root of your project
@@ -44,6 +46,8 @@ You can enable Incremental Static Regeneration ([ISR](https://nextjs.org/docs/ap
   pnpm wrangler kv namespace create NEXT_CACHE_WORKERS_KV
   # or
   yarn wrangler kv namespace create NEXT_CACHE_WORKERS_KV
+  # or
+  bun wrangler kv namespace create NEXT_CACHE_WORKERS_KV
   ```
 
 - Paste the snippet to your `wrangler.toml`:
@@ -73,6 +77,8 @@ Run the following commands to preview the production build of your application l
   pnpm cloudflare
   # or
   yarn cloudflare
+  # or
+  bun cloudflare
   ```
 
 - Preview the app in Wrangler
@@ -83,6 +89,8 @@ Run the following commands to preview the production build of your application l
   pnpm wrangler dev
   # or
   yarn wrangler dev
+  # or
+  bun wrangler dev
   ```
 
 ## Deploy your app
@@ -97,4 +105,6 @@ Deploy your application to production with the following:
   pnpm cloudflare && pnpm wrangler deploy
   # or
   yarn cloudflare && yarn wrangler deploy
+  # or
+  bun cloudflare && bun wrangler deploy
   ```

